csharp/ql/lib/semmle/code/csharp/frameworks/system/Net.qll

```diff
@@ -2,6 +2,7 @@
 
 import csharp
 private import semmle.code.csharp.frameworks.System
+private import semmle.code.csharp.dataflow.ExternalFlow
```

Comment:
Add a `private import semmle.code.csharp.frameworks.system.Net` to the module `Frameworks` in `ExternalFlow.qll` to ensure bidirectional importing.